google-cloud-clients/google-cloud-firestore/src/test/java/com/google/cloud/firestore/LocalFirestoreHelper.java

```diff
@@ -684,7 +697,17 @@ public boolean equals(Object o) {
                 "dateValue",
                 Value.newBuilder()
                     .setTimestampValue(
-                        Timestamp.newBuilder().setSeconds(479978400).setNanos(123000000))
+                        com.google.protobuf.Timestamp.newBuilder()
+                            .setSeconds(479978400)
+                            .setNanos(123000000))
+                    .build())
+            .put(
+                "timestampValue",
+                Value.newBuilder()
+                    .setTimestampValue(
+                        com.google.protobuf.Timestamp.newBuilder()
+                            .setSeconds(479978400)
```

Comment:
The value of seconds is the same as for the server timestamp, but the value of nanoseconds is different. Just checking that it's deliberate.

Reply:
That is deliberate and uses the lowest precision available for each type (note that these constant match the DATE and TIMESTAMP constants in this class). I added comments.